llvm/include/llvm/Target/Target.td

```diff
@@ -583,11 +583,15 @@ class EncodingByHwMode<list<HwMode> Ms = [], list<InstructionEncoding> Ts = []>
   list<InstructionEncoding> Objects = Ts;
 }
 
+class RegClass<RegisterClass Reg, int OperIdx = 0> {
```

Comment:
From a 'coding by example' perspective I wouldn't mind not having a default value for OperIdx